app/assets/stylesheets/_typography.scss

```diff
@@ -1,11 +1,17 @@
 body {
   @include font-feature-settings("kern","liga","frac", "pnum");
-  -webkit-font-smoothing: antialiased;
   background-color: $base-background-color;
   color: $base-font-color;
   font-family: $base-font-family;
   font-size: $base-font-size;
   line-height: $base-line-height;
+  -webkit-font-smoothing: antialiased;
+  
+  #{$all-text-inputs},
+  textarea {
```

Comment:
I think these styles should go inside the same ruleset found in [`_forms.scss`](https://github.com/thoughtbot/bitters/blob/master/app/assets/stylesheets/_forms.scss).